edx_proctoring/templates/proctored_exam/failed-prerequisites.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      This exam is proctored.
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+    This exam is proctored but you have not satisfied the prerequisites for this exam. All requirements for this exam must be satisfactory before you can proceed to take this proctored exam in order to continue to be eligible for credit. You can see the requirements (in order) listed on your <a href="{{progress_page_url}}">Progress</a> page.
+    {% endblocktrans %}
+  </p>
+  <p>
+    {% blocktrans %}
+    The following prerequisites are considered <strong>unsatisfactory</strong>:
+    {% endblocktrans %}
+    <ol style='list-style-type: disc'>
+    {% for requirement in prerequisite_status.failed_prerequisites %}
+      <li>
+        {% if requirement.jumpto_url %}
+          <a href='{{requirement.jumpto_url}}'>{{requirement.display_name}}</a>
+        {% else %}
+          {{requirement.display_name}}
+        {% endif %}
+      </li>
+    {% endfor %}
+    </ol>
+  </p>
+  <p>
+    {% blocktrans %}
+    Given that you have unsatisfactory prerequisites, you currently have no option but to take this exam without proctoring.
```

Comment:
"Due to unsatisfied prerequisites, you can only take this exam without proctoring."